environments/prod/values/hocs-search.yaml.gotmpl

```diff
@@ -1 +1,6 @@
 ---
+hocs-generic-service:
+
+  app:
+    env:
+      sqsEnabled: 'false'
```

Comment:
Is this backwards compatible with 2.82? Because of drone tree this will go to prod on PR merge, so it has to be OK.

Reply:
There is no Search-consumer in prod

Reply:
Let's wait till thats propagated into prod.